src/test/java/org/folio/des/InstallUpgradeIT.java

```diff
@@ -87,7 +89,7 @@ class InstallUpgradeIT {
     new GenericContainer<>(
       new ImageFromDockerfile("mod-data-export-spring").withFileFromPath(".", Path.of(".")))
     .withNetwork(NETWORK)
-    .withExposedPorts(8081)
+    .withExposedPorts(8082)
```

Comment:
@nhanaa , changing 8081 -> 8082 required here as far as you changed this port in application.properties.